### PR TITLE
integration-cli: add cli/api stats tests when container not found

### PR DESCRIPTION
**Pull request by runcom**
kind related to #16361
and making sure we don't regress on this

Signed-off-by: Antonio Murdaca runcom@linux.com

### CODE REVIEW

File: integration-cli/docker_api_stats_test.go [hunk 1]
```diff
@@ -13,7 +14,7 @@ import (
 	"github.com/go-check/check"
 )
 
-func (s *DockerSuite) TestCliStatsNoStreamGetCpu(c *check.C) {
+func (s *DockerSuite) TestApiStatsNoStreamGetCpu(c *check.C) {
```
Comment (jfrazelle) [inline integration-cli/docker_api_stats_test.go]: nice

Comment (runcom) [inline integration-cli/docker_api_stats_test.go]: Ocd is always here

File: integration-cli/docker_cli_stats_test.go [hunk 2]
```diff
@@ -39,5 +39,20 @@ func (s *DockerSuite) TestCliStatsNoStream(c *check.C) {
 		statsCmd.Process.Kill()
 		c.Fatalf("stats did not return immediately when not streaming")
 	}
+}
+
+func (s *DockerSuite) TestStatsContainerNotFound(c *check.C) {
+	testRequires(c, DaemonIsLinux)
```
Comment (duglin) [inline integration-cli/docker_cli_stats_test.go]: I wonder why we need this requirement.

Comment (runcom) [inline integration-cli/docker_cli_stats_test.go]: I saw @jhowardmsft added it to all the other similar tests so I just copy pasted

**Comment by duglin** [comment]
LGTM if janky is happy

**Comment by jfrazelle** [comment]
LGTM, just waiting :)

File: integration-cli/docker_cli_stats_test.go (hunk 2 above)
Comment (jhowardmsft) [inline integration-cli/docker_cli_stats_test.go]: I've turned off almost all CI tests when running against a Windows daemon. Systematically, we're starting turning them on as modifications are done eg #16312

**Pull request merged**